styles/scss/image-gallery.scss

```diff
@@ -137,7 +125,7 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
     content: $ionicon-var-android-contract;
   }
 
-  &.active:hover::before {
+  &.active:hover::before, &.active:focus::before {
```

Comment:
Pretty straightforward, just mirror the hover state w/ the focus state, for keyboard users, on the fullscreen's active state button.